operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3278,6 +3278,8 @@ gateway:
       serverSnippet: ""
       # -- Allows appending custom configuration to the http block
       httpSnippet: ""
+      # -- Allow to set client_max_body_size in the nginx configuration
+      clientMaxBodySize: null
```

Comment:
IMO we should also set it by default. I think the breaking change is worth it given the hard debugging path that you and @mrclrchtr went through. 10M for example won't help a lot with block uploads because it's too low, but it might help with clients sending large requests (like the otel collector)

I also asked @krajorama for an opinion. let's hear others' thoughts are before doing a change to what you have

Reply:
as you mentioned in the other comment, I think 540M would be a good idea

Reply:
let's go with 540M then. Can you make the change?

Reply:
Done

Reply:
Should I now remove the note part? or keep it if somebody changed their configuration?
I think it would be a good idea to keep it but also if somebody changed the default configuration they probably know what they are doing

Reply:
I think it's ok to keep it. More docs rarely hurt :)